AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/SiteAdminController.cs

```diff
@@ -61,7 +61,9 @@ public IActionResult Index()
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> ConfirmDeleteUser(string userId)
         {
-            await _mediator.SendAsync(new DeleteUserCommand { UserId = userId });
+            var user = await _userManager.FindByIdAsync(userId);
+            await _userManager.DeleteAsync(user);

```

Comment:
So...I'm okay doing this, but I think we are still going to need to do a `_mediator.Publish` and punch out an event with the user ID. If we are to 'clean up' in some way when a user is removed (i.e. unassign tasks, remove subscriptions, etc.) we'll need a hook to listen for this type of change. Thoughts